dts/bindings/led/pwm-leds.yaml

```diff
@@ -5,14 +5,27 @@ description: PWM LEDs parent node
 
 compatible: "pwm-leds"
 
+properties:
+    label:
```

Comment:
Can you get this by including base.yaml instead as recommended in the [bindings docs](https://docs.zephyrproject.org/latest/guides/dts/bindings.html#include)?

If you want to just include this property to keep the generated bindings docs clean, you can do:

```yaml
include:
  - name: base.yaml
    property-allowlist: [label]

properties:
  label:
  description: |
    <your description goes here if you need to override the base one>
```

Reply:
Done.